docs/security-features/platform-protections/devices.rst

```diff
@@ -1,28 +1,131 @@
+device to be allowed by existing rules, so while this ``usbauth`` is an effective way to
+minimize the risk external devices connected via USB pose to a user's system, it does not
+outright eliminate said risk.
+
+You can learn more about ``usbauth`` through its 
+`official manpages <https://manpages.ubuntu.com/manpages/noble/en/man8/usbauth.8.html>`_.
 
 
 usbguard
 ========
 
-Starting with Ubuntu 16.10, the usbguard package has been available in universe to provide a tool for using the Linux kernel's USB authorization support, to control device IDs and device classes that will be recognized.
+Starting with Ubuntu 16.10 Yakkety Yak, the ``usbguard`` package has been available in
+universe to provide a framework for implementing USB device policies. It provides a 
+higher degree of security by allowing rules to be created based on a combination of
+device attributes, making it more resilient to devices that provide false information.
+
+The primary tool is the ``usbguard`` command-line utility, which allows a user to generate
+an initial policy, view currently connected devices, and manage rules. The ruleset is stored
+in :file:`/etc/usbguard/rules.conf`. A typical workflow involves running 
+``usbguard generate-policy > /etc/usbguard/rules.conf`` to create a baseline policy that 
+allows all currently connected devices. From that point on, any new device will be blocked by
+default.
+
+When a new device is connected, the ``usbguard`` daemon logs the event, and a user can use 
+``usbguard list-devices`` to see the blocked device. To permanently allow it, a user can use
+``usbguard allow-device <id>`` and then append the new rule to their :file:`rules.conf` file 
+to ensure it persists after a reboot. This makes ``usbguard`` a powerful and dynamic tool 
+for managing USB security on a running system.
+
+While managing ``usbguard`` from the command-line is effective, it can be cumbersome
+on a desktop system. To improve usability, the ``usbguard-notifier`` service is
+available. This tool monitors the ``usbguard`` daemon and provides a desktop
+notification whenever a device is blocked. This notification pop-up allows a user to
+immediately authorize the device and add a permanent rule for it, transforming the
+experience from a manual, command-line process into an interactive one. It is the
+successor to older tools like ``usbguard-applet-qt``.
+
+Included below is an example of the content that may be included in the :file:`rules.conf`
+file after running ``usbguard generate-policy > /etc/usbguard/rules.conf``:
+
+.. code-block:: text
+   :caption: /etc/usbguard/rules.conf
+
+   allow id 046d:c52b name "Unifying Receiver" serial "4071-DE-AD-BE-EF" via-port "usb3-port2" with-interface { 03:01:01 03:01:02 }
+   allow id 046d:082d name "HD Pro Webcam C920" serial "BADA55C0" via-port "usb3-port1" with-interface { 0e:01:00 0e:02:00 }
+   allow id 1d6b:0002 name "Linux Foundation 2.0 root hub" serial "" via-port "" with-interface { 09:00:00 }
+
```

Comment:
I'd give the same general advice for this section as I gave above--if possible, I think it'd be better to give more general context, using the example file for the example variables. This depends on your naming conventions though, which actually look different here than they did in the previous section on usbauth? For example, up there you say that `allow` is the "action", but here you say it's the "target".

So, for example, the first one could be something like:

**Target** (`allow`): This is [definition of "target"]. The example uses `allow`, which means [XX]. Other possible targets are `block` and `reject`.